packages/react/src/analytics/integrations/GTM/eventsMapper.ts

```diff
@@ -23,40 +107,64 @@ const eventsMapper: EventMappers = {
       products: get(properties, 'products', []).map(getProductData),
       list: properties.list,
       currency: properties.currency,
+      from: properties.from,
+      error: properties.error,
+      itemListId: properties.listId,
+      itemListName: properties.list,
+      items: getProductItemsFromEvent(properties),
+      filters: properties.filters
+        ? JSON.stringify(properties.filters)
+        : undefined,
+      sortOption: properties.sortOption,
     };
   },
   [EventType.ProductViewed]: (data: EventData<TrackTypesValues>) => {
     const properties = getEventProperties(data);
+    const items = getProductItemsFromEvent(properties);
 
     return {
       product: getProductData(properties),
+      items,
+      currency: properties.currency,
+      from: properties.from,
+      imageCount: properties.imageCount,
+      value: getEventTotalValue(properties, items),
+      total: properties.total,
     };
   },
   [EventType.ProductClicked]: (data: EventData<TrackTypesValues>) => {
     const properties = getEventProperties(data);
 
     return {
       product: getProductData(properties),
+      from: properties.from,
+      items: getProductItemsFromEvent(properties),
+      item_list_id: properties.listId,
```

Comment:
Why some properties are written as `item_list_id` and others as `itemListId`? Same for `item_list_name` and `itemListName`.

Reply:
Fixed! 👍🏼